app/javascript/components/mapping/MappingForm.jsx

```diff
@@ -144,22 +165,24 @@ const MappingForm = (props) => {
                 It must be a valid URL
               </small>
             </div>
+
             <div className="form-group">
               <label htmlFor="domains_to_map">
-                Which of the following domains are you mapping to?
+                Which domain are you uploading?
               </label>
-              <div className="col-lg-4 col-md-4 domain-option">
-                <div className="form-check-inline">
-                  <label className="form-check-label">
-                    <input
-                      type="checkbox"
-                      value="1"
-                      className="form-check-input"
-                      disabled={submitted}
-                    />
-                    Person
-                  </label>
-                </div>
+              <div className="col-sm-10">
+
+              {domains.map(function (dom) {
+                return (
+                  <div className="form-check" key={dom.id}>
+                    <input className="form-check-input" type="radio" name="domain_id" id={dom.id} value={dom.id} onChange={e => setDomainId(e.target.value)}/>
+                    <label className="form-check-label">
```

Comment:
It'd be nice to have a radio button checked when its label is clicked, i.e. assign `id`s to the inputs and add corresponding `for` attributes to the labels.